newhelm/runners/simple_test_runner.py

```diff
@@ -62,18 +64,17 @@ def run_prompt_response_test(
     annotations_per_annotator: Dict[str, List[Annotation]] = {}
     keyed_annotators = test.get_annotators().items()
     for key, annotator in keyed_annotators:
-        annotations: List[Annotation] = []
-        desc = f"Collection annotations from {key}"
-        for interactions_for_item in tqdm(item_interactions, desc=desc):
-            try:
-                annotation = annotator.annotate_test_item(
-                    interactions_for_item.interactions
+        if use_caching:
```

Comment:
The parallel code between the two blocks, plus the similar branch on caching in _collect_annotations, makes me wonder if there's some way to simplify this. Maybe just by rearranging things. Or maybe, we have a cache interface that has concrete subclasses, one for the sql-on-disk thing, and one for in RAM, or perhaps just a null cache that is always empty.

Reply:
> or perhaps just a null cache that is always empty.

Ooo, I like that. I'll do as a separate PR though since it will impact how we cache SUT responses.